### ISSUE TITLE
[Bug]: Chromium 109.0.5412.0 (r1069273) crashes on M1 Mac when using devtools console

### ISSUE THREAD

**Issue by alexmi256** [puppeteer/puppeteer#9470]
### Bug description

Steps to reproduce the problem:

1. Be on M1 Mac (ARM64)
2. Launch [puppeteer: v19.4.0](https://github.com/puppeteer/puppeteer/releases/tag/puppeteer-v19.4.0) or [puppeteer: v19.4.1](https://github.com/puppeteer/puppeteer/releases/tag/puppeteer-v19.4.1) which use Chromium 109.0.5412.0 (r1069273) without headless mode **OR** download those versions of Chromium from https://storage.googleapis.com/chromium-browser-snapshots/Mac/1069273/chrome-mac.zip or https://storage.googleapis.com/chromium-browser-snapshots/Mac_Arm/1069273/chrome-mac.zip
4. Try to use the write some javascript in the Chromium devtools JS console

Expect:
- Can use devtools JS console

Actual:
- Chromium crashes

Example script:
```.js
import puppeteer from 'puppeteer';

(async () => {
  const browser = await puppeteer.launch({
    ignoreHTTPSErrors: true,
    headless: false,
    devtools: true,
    ignoreDefaultArgs: ['--disable-extensions'],
    args: [
      '--no-sandbox',
      `--ignore-certificate-errors`,
    ],
  });
})();

```

Notes:
- Windows version of chromium seems fine
- Both x86_64 (which uses Rosetta) and ARM64 version of Chromium crash on M1 Mac
- I've tried 19.2.0 which uses 1056772 and this works as expected
- Chromium logs don't appear to give any relevant errors

### Puppeteer version

19.4.1

### Node.js version

19.1.0

### npm version

8.19.3

### What operating system are you seeing the problem on?

macOS

**Comment by OrKoN** [comment]
Able to repro but don't see any crash logs or anything like this. Note that we currently don't officially support ARM and it looks like a problem with the binary so it will be probably solved with the next binary roll.

**Comment by OrKoN** [comment]
We also found a problem the ARM experimental flag: it looks like the flag would lead to the x86 binary download as well. I will give it a try once the new version is published.

**Comment by alexmi256** [comment]
> We also found a problem with the ARM experimental flag: it looks like the flag would lead to the x86 binary download as well. I will give it a try once the new version is published.

AFAICT neither setting `PUPPETEER_EXPERIMENTAL_CHROMIUM_MAC_ARM=1/true` nor `experiments: {macArmChromiumEnabled: true },` works for this

**Comment by OrKoN** [comment]
PUPPETEER_EXPERIMENTAL_CHROMIUM_MAC_ARM is fixed in 19.5.2 but the crash still happens so https://github.com/puppeteer/puppeteer/issues/9470#issuecomment-1378751241 still applies.
